eryx/visuals.py

```diff
@@ -362,3 +364,43 @@ def contribution_curve(self):
             if i_curve == 2:
                 ax.legend(bbox_to_anchor=(2.1,0.5))
         plt.show()
+
+class DeltaPDF:
+
+    def __init__(self, disorder_model, Id=None):
+        self.disorder_model = disorder_model
+        self.q_grid = self.disorder_model.q_grid
+        self.pdf = None
+        if Id is not None:
+            self.Id = Id
+        else:
+            self.Id = self.disorder_model.apply_disorder()
+        self._fill_integral_Miller_points()
+        self._subtract_radial_average()
+
+    def _fill_integral_Miller_points(self):
+        Id_filled = np.copy(self.Id)
+        for q in self.disorder_model._at_kvec_from_miller_points((0, 0, 0)):
```

Comment:
How should this function be applied for non-phonon models?

Reply:
Great point! We now have a more general version when we don't need to "fill" the Bragg intensities
```python
hsampling, ksampling, lsampling = (-7,7,7), (-7,7,7), (-7,7,7)
nidm = NIDM(pdb_path, hsampling, ksampling, lsampling)
nidm_pdf = DeltaPDF(nidm, fill_bragg=False)
nidm_pdf.show(contour=True)
```
![output](https://user-images.githubusercontent.com/4610338/216119330-3adba9a0-9042-4347-9f4e-b876104305a8.png)